src/dotnet/Mgmt.CI.BuildTools/NugetToolsPackage/CI.Tools.Package/nuspec/Microsoft.Internal.AzNetSdk.CITools.nuspec

```diff
@@ -2,7 +2,7 @@
 <package >
   <metadata>
     <id>Microsoft.Internal.NetSdkBuild.Mgmt.Tools</id>
-    <version>0.10.0</version>
```

Comment:
Just so you are aware the nuspect is not currently used any longer I just wanted to keep it around until we have positive confirmation that my new changes fully work. Once we have confirmed that we should delete this file and move the package files under the task project to clean things up a bit.